src/createServer.js

```diff
@@ -1,3 +1,60 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+const createServer = () => {
+  return http.createServer(
+    (req, res) => {
+      const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+      const [convertText, queryString] = req.url.split('?');
+      const originalText = convertText.slice(1);
+      const params = new URLSearchParams(queryString);
+      const targetCase = params.get('toCase');
+      const errorMessages = {
+        noTextMessage: 'Text to convert is required. '
+          + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+        noParam: '"toCase" query param is required. '
+          + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+        invalidCase: 'This case is not supported. '
+          + `Available cases: ${cases.join(', ')}.`,
+      };
+      const errors = [];
+
+      res.setHeader('Content-type', 'application/json');
+
+      if (!originalText) {
+        errors.push({ message: errorMessages.noTextMessage });
+      }
+
+      if (!targetCase) {
+        errors.push({ message: errorMessages.noParam });
+      }
+
+      if (!cases.includes(targetCase) && targetCase?.length) {
```

Comment:
```suggestion
      if (targetCase && !cases.includes(targetCase)) {
```
Just change order of operands to avoid unnecessary array loop.